.vagrant/machines/redis/virtualbox/private_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@olivermel Please make sure to remove all files in the .vagrant folder, and include it in the .gitignore file.